src/components/TopBar/OldUIAnnouncement.tsx

```diff
@@ -0,0 +1,41 @@
+import { Alert, AlertActionCloseButton, AlertVariant } from '@patternfly/react-core';
+import { ApplicationsIcon, ArrowIcon } from '@patternfly/react-icons';
+import { useState } from 'react';
+
+const PNC_OLD_UI_URL_TRIGGER = 'secondary';
+
+export const OldUIAnnouncement = () => {
+  const [isClose, setIsClose] = useState<boolean>(false);
```

Comment:
I think we should use local storage or something to keep the closed state more permanent.

Reply:
If the user closes the announcement and we store the status in local storage, they will not be able to reopen it anymore. Do you think we can add an option in the config toggle of the menu bar for the user to get to the old UI?
![image](https://github.com/user-attachments/assets/4095e280-42f4-4312-bf3f-0fe842d491d7)

Reply:
PM